lib/plugins/compressTimestamp.js

```diff
@@ -62,3 +62,5 @@ exports.update = pluginUtils.createUpdateFilter(fromBasicToExtended, constants.T
 exports.query = pluginUtils.createQueryFilter(fromExtendedToBasic, constants.TIMESTAMP_TYPE);
```

Comment:
CNR entry should be included. But now sure what to say... maybe something like "Re-worked internal architecture (blablabla)", "Major refactor blablabla"... Not sure.